README.md

```
@@ -56,15 +56,21 @@ also known as the _model_.
 
-To do that you can use a dedicated make target:
+No manual steps required beyond merging the model change PR.
+
+### Manual
+
+If the automation is not available, follow these steps:
+
+1. Generate the client types: `make update`
+2. Create a release commit and tags:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)